tests/Microsoft.DotNet.XHarness.Apple.Tests/MockLogs.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using Microsoft.DotNet.XHarness.Common.Logging;
+using Microsoft.DotNet.XHarness.iOS.Shared.Logging;
+using Moq;
+
+namespace Microsoft.DotNet.XHarness.Apple.Tests;
+
+#nullable disable
+public class MockLogs : List<IFileBackedLog>, ILogs
+{
+    public string Directory { get; set; } = "/tmp/logs/";
+
+    public IFileBackedLog AddFile(string path) => AddFile(path, path);
+
+    public IFileBackedLog AddFile(string path, string name)
```

Comment:
A nit: name would be better called description as name in case of file IO usually refers to file name. It's apparent from the line below, but still, would be a tiny bit more readable.

Reply:
The parameter names come from the interface (which is shared with Xamarin). I think it makes more sense to keep them as this is a utility test class?

Reply:
I agree Xamarin chose bad naming though